Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/OverlaySheetService.java

```diff
@@ -779,7 +785,23 @@ public void onFinish() {
     private void showAcknowledgement(String ackType) {
         Handler handler = new Handler(Looper.getMainLooper());
         String ackText = ackType.equals(getString(R.string.DRIVER_ASSIGNMENT)) ? getString(R.string.ride_assigned) : getString(R.string.ride_assigned_to_another_driver);
-        int rawResource = ackType.equals(getString(R.string.DRIVER_ASSIGNMENT)) ? R.raw.ride_accepted_lottie : R.raw.accepted_by_another_driver_lottie;
+        String vehicleVariant = sharedPref.getString("VEHICLE_VARIANT", null);
+        int rawResource;
+        if (ackType.equals(getString(R.string.DRIVER_ASSIGNMENT))){
+            if (key != null && key.equals("yatriprovider") && vehicleVariant.equals("AUTO_RICKSHAW")) {
+                rawResource = R.raw.yatri_auto_accepted_lottie;
+            }
+            else
+                rawResource = R.raw.ride_accepted_lottie;
```

Comment:
by default it would use ride_accepted_lottie which is an auto animation.Can you handle it for other variants also.